solara/website/pages/documentation/components/output/markdown.py

```diff
@@ -43,12 +42,12 @@ def Page():
         with solara.VBox():
             solara.Markdown("# Input text")
             with solara.Padding(2):
-                with rv.Sheet(elevation=2):
-                    rv.Textarea(v_model=markdown_text, on_v_model=set_markdown_text, rows=30)
+                with solara.v.Sheet(elevation=2):
+                    solara.v.Textarea(v_model=markdown_text, on_v_model=set_markdown_text, rows=30)
```

Comment:
We also now have `solara.InputTextArea`, I think it makes sense to replace `solara.v.Textarea` with that.

Reply:
Looks like this wasn't addressed yet.